Automattic/wp-calypso
# Plugins: Update price text to use translate method

## Proposed Changes

Wrap the price and periodicity into a translate method for better i18n and remove the additional blank space.

 
![CleanShot 2023-10-31 at 12 07 05@2x](https://github.com/Automattic/wp-calypso/assets/5039531/5f45cf3f-35d1-4b99-ae76-959f7f3704bc)


## Testing Instructions

* Go to the `/plugins` page 
* Check if the prices on the card are displayed the same way as in production.

## Pre-merge Checklist
